app/boards/arm/sparkfun_micromod_nrf52840/sparkfun_micromod_nrf52840.dts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "sparkfun_micromod.dtsi"
+
+
+/ {
+	model = "Sparkfun MicroMod nrf52840";
+	compatible = "sparkfun,micromod-gpio";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+			label = "Blue LED";
+		};
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&usbd {
+	status = "okay";
+};
+
+/**
+
+&spi1 {
+	compatible = "nordic,nrf-spim";
+	// Cannot be used together with i2c0. 
+	status = "okay";
+	sck-pin = <19>;
+	mosi-pin = <14>; // MOSI = COPI
+	miso-pin = <21>; // MISO = COPO
+	cs-gpios = <&gpio0 32 GPIO_ACTIVE_HIGH>;
+
+
+};
+
+&spi0 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	sck-pin = <28>;
+	mosi-pin = <31>; // MOSI = COPI
+	miso-pin = <2>; // MISO = COPO
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_HIGH>;
+};
+
+**/
+
+
+/**
+&i2c0 {
+	compatible = "nordic,nrf-twi";
+	sda-pin = <8>;
+	scl-pin = <11>;
+};
+
+&i2c1 {
+	compatible = "nordic,nrf-twi";
+	sda-pin = <33>;
+	scl-pin = <24>;
+};
+**/
+
+
+&uart0 {
+	compatible = "nordic,nrf-uarte";
+	tx-pin = <35>;
+	rx-pin = <42>;
+};
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		sd_partition: partition@0 {
+			label = "softdevice";
+			reg = <0x00000000 0x00026000>;
+		};
+		code_partition: partition@26000 {
+			label = "code_partition";
+			reg = <0x00026000 0x000c6000>;
+		};
+
+		/*
+		 * The flash starting at 0x000ec000 and ending at
+		 * 0x000f3fff is reserved for use by the application.
+		 */
+
+		/*
+		 * Storage partition will be used by FCB/LittleFS/NVS
+		 * if enabled.
+		 */
+		storage_partition: partition@ec000 {
+			label = "storage";
+			reg = <0x000ec000 0x00008000>;
+		};
+
+		boot_partition: partition@f4000 {
+			label = "adafruit_boot";
+			reg = <0x000f4000 0x0000c000>;
+		};
```

Comment:
Does the micromod come with the adafruit bootloader/softdevice?

Reply:
We'd need to document how to "prep" a micro mod for use with this board definition then, in the README for this board.